src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/ProjectLaunchTargetsProvider.cs

```diff
@@ -121,16 +104,7 @@ public Task OnAfterLaunchAsync(DebugLaunchOptions launchOptions, ILaunchProfile
 
         public async Task OnAfterLaunchAsync(DebugLaunchOptions launchOptions, ILaunchProfile profile, IReadOnlyList<VsDebugTargetProcessInfo> processInfos)
         {
-            await TaskScheduler.Default;
```

Comment:
Does it make sense to still switch to a background thread here? I assume this is called on the UI thread, and it looks like `ActivateSessionAsync` does some non-trivial work.

Reply:
Ah, yes. That was an accidental removal.